locale/en.po

```diff
@@ -4789,10 +4791,15 @@ msgstr "population:"
 msgid "POPULATION_IN_PATCHES"
 msgstr "population in patches:"
 
-#: ../src/microbe_stage/editor/CellEditorComponent.cs:2323
+#: ../src/microbe_stage/editor/CellEditorComponent.cs:2302
 msgid "POPULATION_IN_PATCH_SHORT"
 msgstr "{0} ({1})"
 
+#: ../src/microbe_stage/editor/MicrobePartSelection.cs:165
+#: ../src/microbe_stage/editor/tooltips/SelectionMenuToolTip.cs:343
+msgid "POSITIVE_SIGNED_NUMBER"
+msgstr "+{0}"
```

Comment:
I actually don't agree with this being made into a translation, mainly because I've used a simple "workaround" in the prototypes already (prepend + when number > 0). This is also not symmetrical with negative numbers as there's no way to override those with a translation.

Reply:
Okay. My solution before this was just to replace - with +, but then I thought about the localization strictness we had.

Reply:
For working with pure numbers, I think it doesn't make sense to add this kind of asymmetry here. Like I said, I'd do something like `"+" + Math.Abs(number).ToString(CurrentCulture)`

Reply:
Changed.